client/my-sites/plans-features-main/index.jsx

```diff
@@ -0,0 +1,111 @@
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PlanFeatures from 'my-sites/plan-features';
+import { PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS } from 'lib/plans/constants';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+
+class PlansFeaturesMain extends Component {
+
+	render() {
+		const { translate, site } = this.props;
+
+		return (
+			<div>
+				<div className="plans-features-main">
+					<PlanFeatures plan={ PLAN_FREE } /* onClick={ this.upgradePlan } */ />
+					<PlanFeatures plan={ PLAN_PREMIUM } /* onClick={ this.upgradePlan } */ />
+					<PlanFeatures plan={ PLAN_BUSINESS } /* onClick={ this.upgradePlan } */ />
+				</div>
+				<FAQ>
+					<FAQItem
+						question={ translate( 'Do you sell domains?' ) }
+						answer={ translate(
+							'Yes! The premium and business plans include a free custom domain. That includes new domains purchased through WordPress.com or your own existing domain that you can map to your WordPress.com site. {{a}}Find out more about domains.{{/a}}',
+							{
+								components: { a: <a href="https://en.support.wordpress.com/all-about-domains/" target="_blank" /> }
+							}
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Can I upload my own plugins?' ) }
+						answer={ translate(
+							'While uploading your own plugins is not available on WordPress.com, we include the most popular plugin functionality within our sites automatically. The premium and business plans even include their own set of plugins suites tailored just for them. {{a}}Check out all included plugins{{/a}}.',
+							{
+								components: { a: <a href={ `/plugins/${ site.slug }` } /> }
+							}
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Can I install my own theme?' ) }
+						answer={ translate(
+							'We don’t currently allow custom themes to be uploaded to WordPress.com. We do this to keep your site secure but all themes in our {{a}}theme directory{{/a}} have been reviewed by our team and represent the highest quality. The business plan even supports unlimited premium theme access.',
+							{
+								components: { a: <a href={ `/design/${ site.slug }` } /> }
+							}
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Do I need another web host?' ) }
+						answer={ translate(
+							'No. All WordPress.com sites include our specially tailored WordPress hosting to ensure your site stays available and secure at all times. You can even use your own domain when you upgrade to the premium or business plan.'
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Do you offer email accounts?' ) }
+						answer={ translate(
+							'Yes. If you register a new domain with our premium or business plans, you can optionally add Google apps for work. You can also set up email forwarding for any custom domain registered through WordPress.com. {{a}}Find out more about email{{/a}}.',
```

Comment:
Copy-wise I would very much like pointing people towards email forwarding.
A huge concern (at least amongst my friends) is having an email with the domain. Basically having domain without email seems wasteful for them.
Google Apps is a great value option but it also is crazy expensive. For the price of Google Apps I can have  a hosting account on other host that provides email.
But then, price-wise I can not pay for wpcom and just install wporg on the host I have to pay for to get email.

The above reasoning is lowering our sales, especially in lower income countries, like Poland.
I'm sure the rich population of Slovakia can afford it though :D

The solution to all of that is to get a free email and use email forwarding.
The downside is that gmail does not allow us to send email from a domain that does not support smtp auth.